digits/scheduler.py

```diff
@@ -208,6 +208,20 @@ def get_job(self, job_id):
             return None
         return self.jobs.get(job_id, None)
 
+    def get_related_jobs(self, job):
+        """
+        Look through self.jobs to try to find the Jobs
+        whose parent contains job
+        """
+        related_jobs = []
```

Comment:
I think this doesn't return anything for model jobs. Should you initialize as `related_jobs = job.parents_job()` to capture the dataset associated with model jobs?